test/e2e/apimachinery/aggregator.go

```diff
@@ -72,13 +72,31 @@ var _ = SIGDescribe("Aggregator", func() {
 		aggrclient = f.AggregatorClient
 	})
 
-	It("Should be able to support the 1.10 Sample API Server using the current Aggregator", func() {
-		// Make sure the relevant provider supports Aggregator
+	// We have to put the version skew test and the storage version hash
+	// test in the same "It" block, because
+	// 1. These two tests require different versions of the sample
+	// apiserver. Otherwise we would be able to setup the server in a
+	// "BeforeEach" block, and run the tests in two different "It" blocks.
+	// 2. We cannot install two sample apiservers at the same time, because
+	// they both serve at "wardle.k8s.io/v1alpha", which is hard-coded.
+	// Otherwise we would be able to run the two tests in two "It" blocks
+	// in parallel.
+	// 3. ginkgo cannot enforce the order of two "It" blocks, unless all
```

Comment:
Maybe we can add a mutex in the two `It` blocks to serialize them, though it will mess up some per `It` block config, e.g., timeout.

Reply:
That won't work because ginkgo launch multiple processes to run subsets of tests.